content/millennium/dstu2/care-provision/goal.md

```diff
@@ -7,6 +7,23 @@ title: Goal | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Goal resource describes intended objectives for a patient.  A Goal is typically expressed as a desired outcome or health state to be achieved by a patient over a period or specific point in time.  A Goal may address preventative health or mitigation of a diagnosis or problem.  Goals can be non-health care related (e.g. dance at a wedding).
```

Comment:
Minor nit: "achieved by a patient over a period or specific point in time." -> "achieved by a patient over a period or *at a* specific point in time." ?